### PR TITLE
fix: upload code cov

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: ci/azure-pipelines.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       - checkout: self
       - script: make unit-test
         displayName: Run unit test
-      - script: bash <(curl https://codecov.io/bash)
+      - script: bash <(curl https://codecov.io/bash) -t 276dd96d-96a9-4518-b7f7-5028a2ddf658
```
Comment (llorllale) [inline ci/azure-pipelines.yml]: can we keep this a secret in the pipeline?

Comment (DRK3) [reply ci/azure-pipelines.yml]: I actually remember reading somewhere that it's possible to do that...but I'm having trouble finding the link...

Comment (DRK3) [reply ci/azure-pipelines.yml]: Although according to the documentation, Azure Pipelines shouldn't even need a token. It's weird that it doesn't work without it anymore.

https://docs.codecov.io/docs/about-the-codecov-bash-uploader

Comment (fqutishat) [reply ci/azure-pipelines.yml]: done

**Pull request merged**